.gitlab-ci.yml

```diff
@@ -60,6 +60,7 @@ variables:
 
   # Publication
   PUBLISH_DOCKER_CLIENT_IMAGES: "false"
+  AUTOMATIC_RELEASE: "false"
```

Comment:
"AUTOMATIC" struck me as a bit odd. Can we call it `PUBLISH_RELEASE` to follow the existing style?

Reply:
Oh now I see this comment should have been in #435.